etc/openqa/openqa.ini

```diff
@@ -78,6 +78,8 @@
 ## Causes jobs reported as incomplete by the worker to be cloned automatically when the
 ## reason matches; set to 0 to disable
 #auto_clone_regex = ^(cache failure: |terminated prematurely: |api failure: Failed to register .* 503|backend died: .*VNC.*(timeout|timed out|refused)|QEMU terminated: Failed to allocate KVM HPT of order 25.* Cannot allocate memory)
+## How many times should a job be restarted
```

Comment:
I guess that should say "maximum amount of times"?

Reply:
`How many times` works, but it shouldn't be inverted:

```suggestion
## How many times a job should be restarted
```

Reply:
done